docs/UserGuide.adoc

```diff
@@ -2357,19 +2363,19 @@ This feature will allow you to import and export your trip data in different for
 |*Mark Packing List Item as Not Packed*: `uncheckitem INDEX [MORE_INDEXES]...`
 |`uncheckitem 1 4 7 13 16`
 
-|*Sort Packing List*: `sortitem CRITERIA ORDER`
+|*Sort Items in Packing List*: `sortitem CRITERIA ORDER`
 |`sortitem name asc`
 
-|*Find Packing List Items by Name*: `finditem KEYWORD [MORE_KEYWORDS]...`
+|*Find Packing List Items by Name*: `finditem NAME [MORE_NAMES]...`
 |`finditem car shampoo`
 
-|*Find Packing List Items by Category*: `finditemcategory KEYWORD [MORE_KEYWORDS]...`
+|*Find Packing List Items by Category*: `finditemcategory CATEGORY [MORE_CATEGORIES]...`
 |`finditemcategory swimming clothes`
 
 |*Clear Packing List*: `clearitem`
 |`clearitem`
 
-|*Add Preset*: `addpreset CATEGORY`
+|*Add Preset*: `addpreset PRESET_NAME`
```

Comment:
I believe the top and in the app is written is `CATEGORY` and I asked Cat yesterday which one to use